tests/integration/test_fim/test_files/test_basic_usage/test_basic_usage_changes.py

```diff
@@ -79,6 +92,8 @@ def test_regular_file_changes(folder, name, encoding, checkers, tags_to_apply,
         Syscheck checkers (check_all).
     """
     check_apply_test(tags_to_apply, get_configuration['tags'])
+    if tags_to_apply == {'ossec_conf_wildcards'}:
+        folder = subdir1 if folder == testdir1 else subdir2
```

Comment:
I don't really like the approach here, maybe we can play around with the parametrize decorators to achieve the same result?